src/Umbraco.Web.UI.Client/src/packages/tiptap/extensions/style-select/style-select-tiptap-toolbar.element.ts

```diff
@@ -0,0 +1,93 @@
+import type { ManifestTiptapToolbarExtension } from '../tiptap-toolbar.extension.js';
+import type { UmbCascadingMenuItem } from '../../components/cascading-menu-popover/cascading-menu-popover.element.js';
+import { css, customElement, html, ifDefined } from '@umbraco-cms/backoffice/external/lit';
+import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+import type { Editor } from '@umbraco-cms/backoffice/external/tiptap';
+
+import '../../components/cascading-menu-popover/cascading-menu-popover.element.js';
+
+@customElement('umb-tiptap-style-select-toolbar-element')
+export class UmbTiptapToolbarStyleSelectToolbarElement extends UmbLitElement {
+	#menu: Array<UmbCascadingMenuItem> = [
```

Comment:
It is interesting to think how we can plug this from outside. Maybe another manifest type or allow existing tiptap extensions to add to it?
We will also need a translator from the rte stylesheets feature over to "tiptap commands".